### PR TITLE
[mysql_service_benchmark] Metadata dict keys to global strings

### PR DESCRIPTION
**Pull request by sspano**
@stfeng @gareth-ferneyhough 
Added global strings for metadata dictionary keys so it is harder to make typos in the future when dealing with metadata keys.

tox passed.

### CODE REVIEW

**Comment by sspano** [comment]
FYI E2E test passed.

File: perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py [hunk 1]
```diff
@@ -131,6 +131,15 @@
 SECONDS_UNIT = 'seconds'
 MS_UNIT = 'milliseconds'
 
+MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
```
Comment (stfeng) [inline perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py]: Is it possible to also use these constants in the flags definition?

e.g., change:
flags.DEFINE_integer('mysql_svc_oltp_table_size', 100000,
                     'The number of rows of each table used in the oltp tests')
to
flags.DEFINE_integer(MYSQL_SVC_OLTP_TABLE_SIZE, 100000,
                     'The number of rows of each table used in the oltp tests')

to do that we probably need to move these string constants definitions up before the flags section.

Comment (sspano) [inline perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py]: Would you like me to move just those necessary variables or all of the variable declarations above flags?
 ie
-----------------
MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
MYSQL_SVC_OLTP_TABLE_SIZE = 'mysql_svc_oltp_table_size'
MYSQL_SVC_DB_INSTANCE_CORES = 'mysql_svc_db_instance_cores'
SYSBENCH_WARM_UP_SECONDS = 'sysbench_warm_up_seconds'
SYSBENCH_RUN_SECONDS = 'sysbench_run_seconds'
SYSBENCH_THREAD_COUNT = 'sysbench_thread_count'
SYSBENCH_LATENCY_PERCENTILE = 'sysbench_latency_percentile'
SYSBENCH_REPORT_INTERVAL = 'sysbench_report_interval'

FLAGS...
BENCHMARK_NAME = 'mysql_service'
BENCHMARK_CONFIG = """
mysql_service:

-------------------
OR 
-------------------
MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
MYSQL_SVC_OLTP_TABLE_SIZE = 'mysql_svc_oltp_table_size'
MYSQL_SVC_DB_INSTANCE_CORES = 'mysql_svc_db_instance_cores'
SYSBENCH_WARM_UP_SECONDS = 'sysbench_warm_up_seconds'
SYSBENCH_RUN_SECONDS = 'sysbench_run_seconds'
SYSBENCH_THREAD_COUNT = 'sysbench_thread_count'
SYSBENCH_LATENCY_PERCENTILE = 'sysbench_latency_percentile'
SYSBENCH_REPORT_INTERVAL = 'sysbench_report_interval'

BENCHMARK_NAME = 'mysql_service'
BENCHMARK_CONFIG = """
mysql_service:
...

FLAGS...

Comment (stfeng) [inline perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py]: moving just those necessary variables will do.

The benefit of doing so is we ensure the name of the flag will always be the same as the labels we put into metadata. We will never misspell.

**Comment by sspano** [comment]
E2E test passed. Can be merged. Please squash commits when merging. Thank you :)

**Pull request merged**